api/envoy/config/route/v3/route_components.proto

```diff
@@ -987,35 +1011,15 @@ message RouteAction {
   // Indicates that the route has a CORS policy.
   CorsPolicy cors = 17;
 
-  // If present, and the request is a gRPC request, use the
-  // `grpc-timeout header <https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md>`_,
-  // or its default value (infinity) instead of
-  // :ref:`timeout <envoy_api_field_config.route.v3.RouteAction.timeout>`, but limit the applied timeout
-  // to the maximum value specified here. If configured as 0, the maximum allowed timeout for
-  // gRPC requests is infinity. If not configured at all, the `grpc-timeout` header is not used
-  // and gRPC requests time out like any other requests using
-  // :ref:`timeout <envoy_api_field_config.route.v3.RouteAction.timeout>` or its default.
-  // This can be used to prevent unexpected upstream request timeouts due to potentially long
-  // time gaps between gRPC request and response in gRPC streaming mode.
-  //
-  // .. note::
-  //
-  //    If a timeout is specified using :ref:`config_http_filters_router_x-envoy-upstream-rq-timeout-ms`, it takes
```

Comment:
Is this feature now going away?

Reply:
Yes, because that header is about the timeout that starts counting from the client's half-close.  gRPC timeouts should always use the timeout that starts counting from the start of the call.

Reply:
Correct.  I've re-added this to the old descriptions, but removed it from the new fields.